test/integration/package.json

```diff
@@ -21,7 +21,8 @@
   "author": "Mr X @Coorp",
   "license": "MIT",
   "engines": {
-    "node": "^18.0.0"
+    "node": "^12.22.12",
```

Comment:
why targeting node 12 ?

Reply:
mmmm, good catch. looks like an oversight!

Reply:
@adamska27 the nvmrc wasn't updated in the node 18. So running code locally  restored the test version.
I just updated the all the version in the test/integration (in [bbfa790](https://github.com/CoorpAcademy/update-node/pull/143/commits/bbfa790fc79241645ff1f5c3fdb543be7bd73f22))